ci/jenkins/test.sh

```diff
@@ -621,6 +627,13 @@ function deliver_antrea {
             scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" flow-aggregator.tar jenkins@[${IP}]:${DEFAULT_WORKDIR}/flow-aggregator.tar
             ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" -n jenkins@${IP} "${CLEAN_STALE_IMAGES}; docker load -i ${DEFAULT_WORKDIR}/antrea-ubuntu.tar; docker load -i ${DEFAULT_WORKDIR}/flow-aggregator.tar" || true
         done
+    elif [[ $TESTBED_TYPE == "kind" ]]; then
+         kubectl get nodes -o wide --no-headers=true | awk '{print $6}' | while read IP; do
+            docker load -i antrea-ubuntu.tar
```

Comment:
I don't think we need this line. The image is built earlier in this script, and antrea/antrea-ubuntu:latest already exist in docker local storage. antrea-ubuntu.tar is saved from antrea/antrea-ubuntu:latest, we don't need to load it again locally.